ore/app/OreApplicationLoader.scala

```diff
@@ -65,12 +66,14 @@ class OreComponents(context: ApplicationLoader.Context)
     with AssetsComponents
     with CaffeineCacheComponents
     with SlickComponents
-    with SlickEvolutionsComponents {
+    with SlickEvolutionsComponents
+    with EvolutionsComponents {
   val prefix                                = "/"
   override lazy val router: Router          = wire[_root_.router.Routes]
   lazy val apiV2Routes: _root_.apiv2.Routes = wire[_root_.apiv2.Routes]
 
   use(prefix) //Gets around unused warning
+  applicationEvolutions
```

Comment:
Can you write something like `eager(applicationEvolutions)`? Same semantics, but makes it more clear that it does stuff at startup.